src/main/java/frc/team1816/robot/commands/LeftAutoStartCommand.java

```diff
@@ -47,17 +46,17 @@ public void selectAuto() {
         } else if (target == 'R'){
             System.out.println("Switch ---- Target: R");
 
-            addParallel(new SetElevatorHeightPercentCommand(20));
+            addParallel(new RaiseCollectorClawCommand(),1);
+            addParallel(new SetElevatorHeightPercentCommand(20),2);
 
             addSequential(new DriveXInchesCommand(210, 0.7));
             addSequential(new RotateXDegreesCommand(90,true,.5),3);
             addSequential(new DriveXInchesCommand(138, 0.7,0.4,0.2));
-            addParallel(new SetElevatorHeightPercentCommand(40));
+            addParallel(new SetElevatorHeightPercentCommand(40),2);
```

Comment:
same issue here with the 40%.  If the encoder isn't working and we go to the top and then rotate, we don't want to tip over.